### PR TITLE
Convert chapters 11 & 15 of Pacemaker Explained to rst

### PR DESCRIPTION
**Pull request by clumens**

### CODE REVIEW

File: doc/sphinx/Pacemaker_Explained/reusing-configuration.rst [hunk 1]
```diff
@@ -1,162 +1,156 @@
...
+        </instance_attributes>
+        <operations>
+          <op id="vm2-monitor-30s" interval="30s" name="monitor" timeout="120s"/>
+          <op id="vm2-stop-0" interval="0" name="stop" timeout="60s"/>
+        </operations>
+      </primitive>
+
+In the example above, the new primitive ``vm2`` has special attribute values.
+Its ``monitor`` operation has a longer ``timeout`` and ``interval``, and
+the primitive has an additional +stop+ operation.
```
Comment (kgaillot) [inline doc/sphinx/Pacemaker_Explained/reusing-configuration.rst:115]: s/+/``/

File: doc/sphinx/Pacemaker_Explained/reusing-configuration.rst [hunk 2]
```diff
@@ -1,162 +1,156 @@
...
+.. code-block:: xml
+
    <rsc_colocation id="vm-template-colo-base-rsc" rsc="vm-template" rsc-role="Started" with-rsc="base-rsc" score="INFINITY"/>
-   
-   would colocate all VMs with +base-rsc+ and is the equivalent of the following constraint configuration:
-   
-   [source,XML]
-   ----
+
+would colocate all VMs with +base-rsc+ and is the equivalent of the following constraint configuration:
```
Comment (kgaillot) [inline doc/sphinx/Pacemaker_Explained/reusing-configuration.rst:150]: s/+/``/

File: doc/sphinx/Pacemaker_Explained/reusing-configuration.rst [hunk 3]
```diff
@@ -238,141 +228,133 @@ Reusing Parts of the Configuration
...
+``nvpair``, and ``operations``.
+
+Tagging Configuration Elements
+##############################
+
+Pacemaker allows you to *tag* any configuration element that has an XML ID.
+
+The main purpose of tagging is to support higher-level user interface tools;
+Pacemaker itself only uses tags within constraints. Therefore, what you can
+do with tags mostly depends on the tools you use.
```
Comment (kgaillot) [inline doc/sphinx/Pacemaker_Explained/reusing-configuration.rst:307]: Might as well update this section to reference crm_mon --resource/--node (probably in its own commit)

File: doc/sphinx/Pacemaker_Explained/multi-site-clusters.rst [hunk 4]
```diff
@@ -1,345 +1,342 @@
...
+
+      # crm_ticket --info
+
+Or you can monitor them with:
+
+   .. code-block:: none
+
+      # crm_mon --tickets
+
+Display the +rsc_ticket+ constraints that apply to a ticket:
```
Comment (kgaillot) [inline doc/sphinx/Pacemaker_Explained/multi-site-clusters.rst:310]: s/+/``/

File: doc/sphinx/Pacemaker_Explained/multi-site-clusters.rst [hunk 5]
```diff
@@ -1,345 +1,342 @@
...
+   .. code-block:: none
+
+      # crm_ticket --ticket ticketA --constraints
+
+When you want to do maintenance or manual switch-over of a ticket,
+revoking the ticket would trigger the loss policies. If
+``loss-policy="fence"``, the dependent resources could not be gracefully
+stopped/demoted, and other unrelated resources could even be affected. 
+
+The proper way is making the ticket 'standby' first with:
```
Comment (kgaillot) [inline doc/sphinx/Pacemaker_Explained/multi-site-clusters.rst:321]: s/'/*/

**Review by kgaillot — CHANGES_REQUESTED** [review]
I can see the light at the end of the tunnel :)

**Comment by clumens** [comment]
Yep, we're getting close on it.  Only a couple chapters left and we can stop worrying about sphinx.

**Review by kgaillot — APPROVED** [review]

**Pull request merged**